### DISCUSSION TITLE
Charging curve stats issue (maybe a bug?)

### DISCUSSION THREAD

**Discussion by djpalmis** [jheredianet/Teslamate-CustomGrafanaDashboards#44] [General]
Strange behaviour in the charging curve stats dashboard. 
Yesterday I DC charged 2 times. 
Official teslamate can show the sessions and the curves.
Even your dashboard can show the max power graph but not the curve. 
What do you think is happening? 
![Screenshot_20230824_155447_Samsung Internet](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/402122/18204a30-e217-4350-9b2d-a355d034e613)

**Comment by jheredianet** [comment]
Yes, it's strange. No idea what is happening. 
May be you can look futher on the query. Try to edit the charging curve panel and use the "query inspector" to get que final query it uses to get the data. 

![image](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards/assets/45735191/032a6201-f4fd-4d37-abf3-f5a533ec4c82)

Then you can, run that query outside Grafana and try to see why it is not getting records.

**Reply by djpalmis** [reply]
Sorry: not skilled guy here. Can you explain me how run the query outside grafana?

**Comment by jheredianet** [comment]
Basically, you should have a direct connection with your postgresql Database using any cliente like pgAdmin or DBeaver to browse your tables and run queries. Grafana Explore tool will do as well (although it's some kind limited).

Did you try to change the range of the dates? It could help you to detect exactly where it stops working so with the query you may investigate further.

**Comment by djpalmis** [comment]
> Did you try to change the range of the dates? It could help you to detect exactly where it stops working so with the query you may investigate further.

Sorry: What i was able to do is to confirm that both charging session (in the same day) gave problem (i played with dates).
Honestly I can't find a way to debug the issue.
I'm sorry. :(
